interface/code_systems/cqm_valueset_howto.php

```diff
@@ -28,7 +28,7 @@
 <div class="dialog"><p>
 <?php echo xlt("Steps to install the VALUSET database"); ?>:
 <ol>
-<li><?php echo xlt("The first step is to download the VALUESET release. Access to VALUESET is provided by NLM. Only valueset for Eligible Professionals need to be downloaded and it should be downloaded in XML format from Sorted By CMS ID column. For more details see the below link") .
+<li><?php echo xlt("The first step is to download the VALUESET release. Access to VALUESET is provided by NLM. Only valueset for Eligible Professionals need to be downloaded and it should be downloaded in XML format from Sorted By Payer ID column. For more details see the below link") .
 " <a href='https://vsac.nlm.nih.gov/#download-tab'>https://vsac.nlm.nih.gov/#download-tab</a>."; ?> 
```

Comment:
hi @juggernautsei , the cms id here is not referencing the payer id. cms id is ok for the cqms

Reply:
Ok, I have reverted that change.